spark/src/main/resources/python/zeppelin_pyspark.py

```diff
@@ -107,6 +107,50 @@ def isAutoConvertEnabled(self):
   def isImportAllPackageUnderSparkSql(self):
     return self.version >= self.SPARK_1_3_0
 
+class PySparkCompletion:
+  def getGlobalCompletion(self):
+    objectDefList = []
+    try:
+      for completionItem in list(globals().iterkeys()):
+        objectDefList.append(completionItem)
+    except:
+      return None
+    else:
+      return objectDefList
+
+  def getMethodCompletion(self, text_value):
+    objectDefList = []
+    completion_target = text_value
+    try:
+      if len(completion_target) <= 0:
+        return None
+      if text_value[-1] == ".":
```

Comment:
@felixcheung  
Thank you for Feed back.
You need to filter the __ private method?
I think filtering is unnecessary for __.
This is because in the private method __ to think you can use a class that you have created.
If you ever need a filter, you can hear more reasons?

- my English is not very good. Thank you good feedback -